### DISCUSSION TITLE
Getting input on future releases

### DISCUSSION THREAD

**Discussion by birtles** [birchill/10ten-ja-reader#1466] [General]
Hi!

I've just sent version 1.16 off to the different app stores so it should start showing up in the next week or so, depending on your browser.

The main change is switching to MV3 on Chrome / Edge and to non-persistent event pages on Firefox / Thunderbird. Hopefully there aren't too many nasty regressions from doing that. Please tell me if you notice anything odd though.

I've also tried to address issues that affect Firefox for Android since they are rolling out full support for Web extensions in December.

In future I'd like to do a better job at being transparent about upcoming releases and getting feedback about what people most want to see fixed. To that end I've set up a meta-issue for tracking the next release here: https://github.com/birchill/10ten-ja-reader/issues/1465

Please submit feedback there about bugs you really want to see fixed in the next release.

I realise, however, that most people using 10ten are not using GitHub or even X/Twitter so in the new year I hope to find a way to communicate changes more broadly and get more input. That will probably start with a blog of some sort but ideally will also grow either a discussion forum or a Discord server or both.

I also hope to blog about the bigger changes I'm planning to tackle this coming year which, in summary, is something like:

- Monolingual dictionary support
- Flashcard generation support
- Lookup bar support (or something equivalent)
- Better opt-in permissions support

Thanks for your patience with these. I'm working on this stuff roughly full-time 6 days a week but doing it well takes time. (And if anyone is in Tokyo looking for an internship or elsewhere looking to do some remote contract work let me know!)

**Comment by StarScape** [comment]
> Flashcard generation support

I'm a few months late here, but this is a feature I'm interested in, if there's any way I could help out. I currently use Yomichan on desktop because I like to occasionally make Anki flashcards from new words. But I'd happily switch to 10ten on all platforms if it had equivalent support—and I'd *kill* for flashcard generation support on mobile, though that might be a bit trickier.

**Reply by birtles** [reply]
Definitely in the works!
